src/azure-cli/setup.py

```diff
@@ -139,6 +139,8 @@
     'azure-synapse-spark~=0.2.0',
     'chardet~=3.0.4',
     'colorama~=0.4.4',
+    # On Linux, the distribution (Ubuntu, Debian, etc) and version are checked for `az feedback`
+    'distro; sys_platform == "linux"',
```

Comment:
This is what it becomes in generated wheel `METADATA`:

```
Requires-Dist: distro ; sys_platform == "linux"
```

Reply:
You should use `platform_system == "Linux"` .

Here: https://peps.python.org/pep-0508/#environment-markers
It says that `sys_platform` will be linux in python3 and linux2 in python2

Reply:
It's ok. We have dropped support for Python 2 anyway.

We also use `sys.platform` elsewhere:

https://github.com/Azure/azure-cli/blob/fb805f4bb3167623107c10ec5f34581a838c8a1c/src/azure-cli-core/azure/cli/core/auth/persistence.py#L41-L50

Reply:
`platform.system()` looks weird on cygwin, and `sys.platform` seems to be easier:

```
$ python3 -c "import os; print(os.name)"
posix

$ python3 -c "import sys; print(sys.platform)"
cygwin

$ python3 -c "import platform; print(platform.system())"
CYGWIN_NT-10.0-19044
```

Reply:
Ok sounds good to me